blockexplorer/src/app/components/Footer.tsx

```diff
@@ -61,9 +61,9 @@ export function Footer() {
             >
               <Grid item>
                 {theme.palette.mode === 'light' ? (
-                  <DarkLogo style={{ height: '40px', width: 'auto' }} />
```

Comment:
This file has been moved to the host app.

Reply:
I have performed a merge, there should be no problem now